demisto_sdk/tests/test_files/valid-beta-integration.yml

```diff
@@ -1,6 +1,6 @@
 category: Messaging
 commonfields:
-  id: Telegram_Beta
+  id: Telegram
```

Comment:
This is for testing right?

Reply:
right

Reply:
If we are done testing, let's revert this.